Gridy/View/whiteView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  GridOverlay.swift
+//  Gridy
+//
+//  Created by Alexandra Ivanova on 29/10/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+class whiteView: UIView {
```

Comment:
Weird name and it doesn't start with an uppercase

Reply:
fixed 👍